src/app/components/FormElements/FormFieldWrapper.tsx

```diff
@@ -2,19 +2,20 @@ import * as React from 'react';
 import { FormGroup, ControlLabel, HelpBlock } from "react-bootstrap";
 import { WrappedFieldProps } from 'redux-form';
 
-export interface FormFieldWrapperProps extends WrappedFieldProps {
-    label: string;
+export interface FormFieldWrapperProps extends WrappedFieldProps{
+    label:string;
+    showControlLabel?: boolean;
```

Comment:
how would you feel about **`showLabel`** vs **`showControlLabel`**

Reply:
That makes sense - makes it a little more generic